configs/config.js

```diff
@@ -49,6 +49,7 @@ const OWASP_CONFIG = {
 
 const WORDPRESS_CREDENTIALS = {
   url: 'https://www.iee.ihu.gr',
+  is_multilingual: false, 
```

Comment:
The code style convection we follow is camel cased, so please try follow this for consistency.
In this case "is_multilingual" shloud be "isMultilingual"